src/app/malaria-101/malaria-101.component.scss

```diff
@@ -0,0 +1,238 @@
+@import "../styles/main";
+
+video {
+    border: 3px solid $grey;
+    background-color: $black !important;
+}
+
+.pieces {
+    height: 125px;
+    width: 115px;
+    border-radius: 25px;
+}
+
+.game-container {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+}
+
+.lifecycle-imgs {
+    max-width: 100%;
+    max-height: 100%;
+    margin: auto;
+}
+
+@mixin text($topPos, $leftPos) {
+    position: absolute;
+    width: 15%;
+    top: $topPos;
+    left: $leftPos;
+}
+
+@mixin rotate($rotation) {
```

Comment:
don't we have a `rotate` mixin elsewhere?

actually this entire file feels duplicated